gitplay/.gitignore

```diff
@@ -0,0 +1,44 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Seems like you picked up a template. Nice!!

Reply:
This was not me. This file was auto generated by Angular. But now that you have pointed it out. I will try to understand it.